community/artifactory-ha/templates/artifactory-primary-statefulset.yaml

```diff
@@ -25,15 +25,14 @@ spec:
         role: {{ template "artifactory-ha.primary.name" . }}
         component: {{ .Values.artifactory.name }}
         release: {{ .Release.Name }}
-        heritage: {{ .Release.Service }}
-        chart: {{ template "artifactory-ha.chart" . }}
```

Comment:
Please add `heritage` and `chart` values back unless there is a compelling reason to remove them.